plugins/obs-outputs/librtmp/parseurl.c

```diff
@@ -276,4 +276,5 @@ void RTMP_ParsePlaypath(AVal *in, AVal *out)
 
     out->av_val = streamname;
     out->av_len = destptr - streamname;
+    free(streamname);
```

Comment:
Pretty sure this fix isn't correct; `streamname` is still being referenced as `out->av_val`, so it has to be `free`d elsewhere

Reply:
ah shit, ok i'll revert it. Thanks for the comment